export.py

```
@@ -70,67 +72,102 @@ def login(creds, captcha):
         time.sleep(LOGIN_DELAY)
 
     try:
-        user = driver.find_element(By.NAME, 'USER')
+        user = driver.find_element(By.NAME, 'latitude-id-email-address')
     except NoSuchElementException as exception:
         exit("Could not find the login screen. Use --captcha option if you need to manually complete a captcha")
 
     user.send_keys(creds[0])
-    user = driver.find_element(By.NAME, 'PASSWORD')
+    btn = driver.find_element(By.CSS_SELECTOR, "div[class*='login_container'] div[class*='loginWidget'] button")
+    btn.click()
+    time.sleep(LOGIN_DELAY)
+    user = driver.find_element(By.NAME, 'password')
     user.send_keys(creds[1])
-    btn = driver.find_element(By.NAME, 'SUBMIT')
+    btn = driver.find_element(By.CSS_SELECTOR, "#loginScrollableContainer button > span")
+
+    # btn = driver.find_element(By.NAME, 'SUBMIT')
     btn.click()
 
     time.sleep(WAIT_DELAY)
 
-    tranLink = driver.find_element(By.XPATH, u'//a[text()="View Transactions"]')
+    try:
+        terms_link = driver.find_element(By.CSS_SELECTOR, '#loginScrollableContainer > div > div > section > button')
+        terms_link.click()
+    except Exception as exc:
+        pass
```

Why do we ignore all exceptions?